sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -107,6 +107,7 @@ statement
     : query                                                            #statementDefault
     | ctes? dmlStatementNoWith                                         #dmlStatement
     | USE NAMESPACE? multipartIdentifier                               #use
+    | SET CATALOG multipartIdentifier                                  #setCatalog
```

Comment:
multipartIdentifier? I think it should be `identifier | STRING`, so that people can write `SET CATALOG abc` or `SET CATALOG '_  56'`

Reply:
+1

Reply:
Thank you. I get it.